integration-tests/fake_spine/Pipfile

```diff
@@ -6,6 +6,7 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
+integration-adaptors-common = {editable = true,path = "./../../common"}
```

Comment:
Should it be modified here or you've just forgot to add -e when installing the package?

Reply:
It is my understanding that module dependencies should be included in the Pipfile